internal/backends/collection.go

```diff
@@ -227,15 +227,16 @@ type CollectionStatsParams struct {
 
 // CollectionStatsResult represents the results of Collection.Stats method.
 //
+// CappedSize and CappedDocuments are zero for not capped collection.
 // CountObjects is an estimate of the number of documents.
-//
-// TODO https://github.com/FerretDB/FerretDB/issues/2447
 type CollectionStatsResult struct {
-	CountObjects   int64
-	CountIndexes   int64
-	SizeTotal      int64
-	SizeIndexes    int64
-	SizeCollection int64
+	CappedSize      int64
+	CappedDocuments int64
+	CountObjects    int64
```

Comment:
What is the difference between those two?

And let's finally rename "objects" to "documents"

Reply:
renamed, also getting capped document info from ListCollections now